ibm/resource_ibm_cis_page_rule.go

```diff
@@ -415,6 +416,10 @@ func flattenCISPageRuleActions(actions []cispagerulev1.PageRulesBodyActionsItemI
 			value := item.Value.(map[string]interface{})
 			actionItemOutput[cisPageRuleActionsValueURL] = value[cisPageRuleActionsValueURL]
 			actionItemOutput[cisPageRuleActionsValueStatusCode] = value[cisPageRuleActionsValueStatusCode]
+		} else if *item.ID == cisPageRuleActionsIDBrowserCacheTTL || *item.ID == cisPageRuleActionsIDEdgeCacheTTL {
```

Comment:
issue was describing that it was having issues with `minify` as well.. Is that resolved?
Can we get test results of below steps attached to this PR?
```
Import a page rule with either a minify or browser_cache_ttl action.
terraform state show ibm_cis_page_rule.imported_page_rule and see that no actions exist.
```

Reply:
@kavya498 
The issue with minify is resolved in latest commit.

Test results:
Screenshot of `terraform state show ibm_cis_page_rule.imported_page_rule` result.

<img width="1440" alt="Screenshot 2021-12-06 at 2 21 40 PM" src="https://user-images.githubusercontent.com/79745361/144823611-10afcd6c-d367-4374-acdd-9268738568e1.png">